packages/ui/src/Dialog/index.tsx

```diff
@@ -66,6 +50,8 @@ const DialogContent = styled(RadixDialog.Content)`
   display: flex;
   flex-direction: column;
   gap: ${props => props.theme.spacing(6)};
+
+  pointer-events: auto;
```

Comment:
We had to add `pointer-events: auto` here because we're using `pointer-events: none` on `DialogContent` above, so that clicks on `DialogContent` are passed through to the `Overlay` and thus close the dialog.